src/mumble/CMakeLists.txt

```diff
@@ -514,9 +514,9 @@ if(bundled-opus)
 		install(TARGETS opus LIBRARY DESTINATION lib)
 	endif()
 else()
-	find_pkg(Opus CONFIG REQUIRED)
-	add_dependencies(mumble PRIVATE Opus::opus)
-	target_include_directories(mumble PRIVATE ${Opus_INCLUDE_DIRS})
+	find_pkg(opus CONFIG REQUIRED)
+	target_link_libraries(mumble PRIVATE ${opus_LIBRARIES})
```

Comment:
@davidebeatrici you said that we don't have to link Opus. In that case though I think we can't do anything here as we don't have an Opus target available but `add_dependencies` only worls with those :thinking:

Reply:
IIRC from my tests yesterday, it is needed as it will complain about missing symbols when linking otherwise.

Reply:
And using
```
add_dependencies(mumble PRIVATE opus::opus)
```
didn't work?

Reply:
That is correct.

This is because there is no opus target when not using the bundled opus library.

Reply:
As mentioned before the mumble-git package for Arch Linux uses this [patch](https://aur.archlinux.org/cgit/aur.git/tree/no-bundled-opus.patch?h=mumble-git) (without `target_link_libraries`).
It works fine here and should also work on Gentoo. @DarkDefender: Please try it on your system.

Reply:
Specifying `CONFIG` is actually not required:
https://github.com/mumble-voip/mumble/blob/0ba1eac51b5f4443e6fbb1bc3e157403903e9a01/cmake/pkg-utils.cmake#L77-L80

`find_pkg()` attempts to find the package with `CONFIG` and then `MODULE`.

Reply:
Check and you guys are indeed correct, I've updated the patch.

Thanks for the rigorous testing :+1: